SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts

```diff
@@ -155,11 +169,14 @@ export class PixiTileMapHelper {
       } else if (layer instanceof EditableTileMapLayer) {
         const tileLayer = layer as EditableTileMapLayer;
 
+        const tileWidth = tileLayer.tileMap.getTileWidth();
+        const tileHeight = tileLayer.tileMap.getTileHeight();
+
         for (let y = 0; y < tileLayer.tileMap.getDimensionY(); y++) {
+          let first = true;
```

Comment:
this seems unused?

Reply:
oh, yes, it was for debugging.
good catch